src/CMakeLists.txt

```diff
@@ -696,9 +696,11 @@ install(PROGRAMS
 
 add_subdirectory(bash_completion)
 
-set(parse_secret_files
-  common/secret.c)
-add_library(parse_secret_objs OBJECT ${parse_secret_files})
+if(LINUX)
```

Comment:
@wjwithagen could you please note why this is not `secret.c` is not buildable in FreeBSD? for example, keyutils is not available in FreeBSD?

and why krbd is not applicable to FreeBSD?